robusta_krr/core/integrations/prometheus/metrics/base_metric.py

```diff
@@ -39,7 +39,7 @@ def get_prometheus_cluster_label(self) -> str:
         """
         if self.config.prometheus_cluster_label is None:
             return ""
-        return f', cluster="{self.config.prometheus_cluster_label}"'
+        return f', {self.config.prometheus_label}="{self.config.prometheus_cluster_label}"'
```

Comment:
It seems that `prometheus_label` is optional. but you use it here (it will be `", None=..."`)

Reply:
good call, it has a default value so it will never be none